audit/projects/k8s-staging-autoscaling/iam.json

```diff
@@ -40,7 +40,6 @@
     {
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
-        "group:k8s-infra-release-viewers@kubernetes.io",
```

Comment:
Why was this removed?

Reply:
So these was removed by implementing thix fix: https://github.com/kubernetes/k8s.io/pull/756/commits/88c811aa1803cda20e8f1a248515065d80a3a362